### ISSUE TITLE
[BUG] release source archive contains version `0.11.1-alpha.0` rather than 0.11.0

### ISSUE THREAD

**Issue by chenrui333** [Nukesor/pueue#169]
👋  it looks like the release source archive has unmatched releaseVersion `0.11.1-alpha.0` rather than `0.11.0`. Thanks!

**Comment by Nukesor** [comment]
Ah damn it. You're right.
I changed the project structure and forgot to adjust the Github release action, which failed right away.

A new patch (v0.11.1) has been released to properly fix this.


Turns out cargo-release really isn't optimized for working with workspaces...
The next development cycle always starts with `{patch+1}-alpha.0`.
Since this version is an `alpha`, the relative path version from `pueue` to `pueue-lib` must be pinned to this exact version (because alpha).

But when publishing to crates.io, the relative path version doesn't get adjusted and the release fails half-way through.
The release for `pueue-lib` finishes to upload to crates-io, but the tag isn't created and no commits are pushed.
The release fails because cargo cannot find the `alpha.0` version on crates.io (naturally).

I wish there was a way to skip setting the next development cycle with cargo-release.
At least I couldn't find it.
